tests/playbooks/tests_mac_address_match.yml

```diff
@@ -0,0 +1,43 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- name: Play for testing MAC address match on device
+  hosts: all
+  vars_prompt:
+    - name: "interface"
+      prompt: "Please provide the interface name (e.g., eno2)"
+      private: no
+    - name: "mac"
+      prompt: "Please provide the MAC address to match (e.g., ec:f4:BB:d3:EC:92)"
+      private: no
+  vars:
+    profile: "{{ interface }}"
+    vlan_profile: "{{ interface }}.3732"
+    lsr_fail_debug:
+      - __network_connections_result
+  tags:
+    - "tests::mac"
```

Comment:
@spetrosi I think we'll need to exclude this test from automated testing - we'll need to add `tests::mac` to the `--skip-tags` list  @liangwen12year I think this test is meant to be run interactively

Reply:
Created a PR for that https://github.com/linux-system-roles/tft-tests/pull/65

Reply:
I don't follow why the test cannot determine the MAC address itself instead of asking for it in a prompt.

Reply:
The test is supposed to operate on an interface which has the permanent address, I do not make the test accidentally configured on the interface that is used to bridge the ssh connection between the control node and managed host.  Possibly, I can search through all the links in `/sys/class/net/` and use the first unconfigured ethernet interface. But such an ethernet interface (unconfigured and also have a permanent mac address) may not exist in the system environment of upstream testing.

Reply:
You are right. We can use `ethtool -P $iface` to determine the mac address of the interface.

Reply:
We controll the upstream test envirionment, so it should be clear whether or not such an interface exists and if it does not, there might be an option to create such an interface.

Nevertheless, the prompt also does not specify the requirements for the interface to ensure that it is eligible/the test does not ensure that it actually tests what it is supposed to test.

Reply:
Honestly, it turns out that the upstream testing environment does not have the interface with the permanent mac address.  I also tried `iproute2` , `systemd-networkd`, and NetworkManager, but I can not create such an interface with the permanent mac address (the permanent mac address can be queried by calling `ethtool -P $iface` ), I can only create an interface with the current mac address (the current mac address can be queried by calling `cat /sys/class/net/$iface/address` ).

Reply:
I specified the requirements for the interface to ensure that it is eligible.

Reply:
How does the upstream environment work?

Reply:
Summarized from the slack discussion, the upstream testing uses Testing Farm, which uses AWS machines, we do not run kvm/qemu virtual machines on the AWS machines. According to the doc https://tmt.readthedocs.io/en/stable/spec/hardware.html#network, requesting a machine with extra NICs should be already supported via `network.type`, we should be able to provision a machine with 2 NICs by specifying the following in the network system role test configuration. 

```
hardware:
  network:
    - type: eth
    - type: eth
```